pages/_app.tsx

```diff
@@ -23,9 +24,9 @@ export default function MyApp({ Component, pageProps }: AppPropsWithLayout) {
   //eslint-disable-next-line react/jsx-props-no-spreading
   const layout = getLayout(<Component {...pageProps} />, session);
   return (
-    <>
+    <ErrorBoundary>
       <BrowserIsOutdatedBanner />
       {layout}
-    </>
+    </ErrorBoundary>
```

Comment:
I fear this might break stuff, especially the redirect on error logic

Reply:
Only on client. And that's the goal, we want to differentiate between 500 errors (serverside) and client error :
- On client, add other metadata (like component call stack)
- It should fix the reporting of error, which is broken on the client 

See: 
- [Error that are not properly catched](https://errors.data.gouv.fr/organizations/sentry/issues/129277/?project=15&query=is%3Aunresolved+assigned%3Ame&referrer=issue-stream&statsPeriod=14d)
- [What next does say about them](https://nextjs.org/docs/messages/client-side-exception-occurred)
- [What is the recommanded way in the doc](https://nextjs.org/docs/pages/building-your-application/configuring/error-handling)
- « Error boundaries do not catch errors for Server side rendering » in [react documentation](https://legacy.reactjs.org/docs/error-boundaries.html)